blog/_posts/2017/qa_webcompat/2018-01-14-MozillaIN-QA-Contributions-2017.markdown

```diff
@@ -85,7 +98,9 @@ October was a very special month for mozilla, every mozillian consistently worke
 
 In the end of the month we had a [QA call](https://fahimazulfath.wordpress.com/2017/12/24/qa-test-day-online-event/) for newbies to help them participating in QA Test Days.  In case you missed the call. you can watch the video in [YouTube](https://www.youtube.com/playlist?list=PLV5zqYRQOU3lJ1zWsLS9OUYYjmHtkhc0x&jct=ijtCDC9slLtDSnHkyt-t7bKGgMvezA). 
  
-That was an amazing time we had together as a community in the proud name of **Mozilla India**. Words are powerless to express the gratitude for the community members for taken initiatives to achieve this metrics and for supporting us in all aspects. Last but not least Mozilla India would like to thank **Mr. Mihai Boldan**, **Mr. Mike Taylor** and **Mr. pasal Chevrel** for mentoring us. Many thanks to [Mr. Prasanth](https://twitter.com/prasanthp96), [Mr. Surentharan](https://twitter.com/surentharan7), [Ms. Fahima Zulfath](https://twitter.com/FahimaZulfath) and [Ms. Krishnaveni](https://twitter.com/venikrishbm) for supporting us on organizing the bug events. We are looking for tremendous powerful contributions from all of you in upcoming QA days.
+That was an amazing time we had together as a community in the proud name of **Mozilla India**. Words are powerless to express the gratitude for the community members for taken initiatives to achieve this metrics and for supporting us in all aspects. Last but not least Mozilla India would like to thank **Mr. Mihai Boldan**, **Mr. Mike Taylor** and **Mr. pasal Chevrel** for mentoring us. 
```

Comment:
add mozillians profile link for Mozilla employees. "Pascal Chevrel"  check the uppercases.